decentraland/unity-renderer
# feat: add zoom in and zoom out to map

## What does this PR change?

Fix #1315
Adds the possibility of zooming in and out in the map with the mouse scrollwheel or with the +/- keyboard buttons

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/zoom-in-map
2. Open the map
3. Zoom in and out with scrollwheel
4. Zoom in and out with +/- keyboard buttons
5. Verify behavior when opening and closing the map

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
